app/views/pet_applications/show.html.erb

```diff
@@ -2,3 +2,5 @@
 <p>Name: <%= @pet_application.name %></p>
 <p>Address: <%= @pet_application.street %>, <%= @pet_application.city %>, <%= @pet_application.state %>, <%= @pet_application.zip %></p>
 <p>Description: <%= @pet_application.description %></p>
+
+<%= link_to @pet.name, "/pets/:id" %>
```

Comment:
Figuring out how to get the params/variables to function here is a weakness of mine. This isn't correct.